src/api/checkout/checkout_payment_activations_auth/v1/_swagger.json.tpl

```diff
@@ -1,7 +1,7 @@
 {
   "swagger": "2.0",
   "info": {
-    "version": "1.0.0",
+    "version": "1.1.0",
```

Comment:
v1 auth APIs do not need to be updated, they are used by current versions of IO app